lr-cli/cortx_setup/commands/storageset/config/durability.py

```diff
@@ -64,52 +73,71 @@ class DurabilityConfig(Command):
 
     def run(self, **kwargs):
         try:
-            index = 'storage_durability_index'
+            confstore_file_index = 'storage_durability_index'
             storage_set_name = kwargs['storage_set_name']
             durability_type = kwargs['durability_type']
             cluster_id = get_cluster_id()
+            durability_dict = {
+                'data': str(
+                    kwargs['data']), 'parity': str(
+                    kwargs['parity']), 'spare': str(
+                    kwargs['spare'])}
 
             self.load_conf_store(
-                index,
+                confstore_file_index,
                 f'json://{CONFSTORE_CLUSTER_FILE}'
             )
 
-            ss_name = Conf.get(index,
-                               f'cluster>{cluster_id}>storage_set[0]>name')
+            current_storage_set_data = Conf.get(
+                confstore_file_index, f'cluster>{cluster_id}>storage_set')
+            storage_index = -1
 
-            if ss_name != storage_set_name:
+            for idx, _ in enumerate(current_storage_set_data):
+                if current_storage_set_data[idx]['name'] == storage_set_name:
+                    storage_index = idx
+                    break
+
+            if storage_index == -1:
                 raise ValueError(
-                   "Invalid Storageset name provided: "
-                   f"'{storage_set_name}' not found in ConfStore data. "
-                   "First, set with `cortx_setup storageset create` command."
+                    "Invalid Storageset name provided: "
+                    f"'{storage_set_name}' not found in ConfStore data. "
```

Comment:
Remove keyword ConfStore... Write simple error message as follows:
`Error: Storage-set '{storage_set_name}' not found`.
 `Action: Configure with cortx_setup storageset create command`

Reply:
Updated